Controller/app-controller.js

```diff
@@ -1,7 +1,16 @@
-const {selectTopics} = require('../Model/app-model')
+const {selectTopics, selectArticlesById} = require('../Model/app-model')
 
```

Comment:
Picky picky but we don't expect to select articles plural by id, but a single article. Your naming should reflect this.